src/agent/coverage/Cargo.toml

```diff
@@ -26,3 +26,4 @@ procfs = { version = "0.12", default-features = false, features=["flate2"] }
 clap = { version = "4.0", features = ["derive"] }
 env_logger = "0.10.0"
 pretty_assertions = "1.3.0"
+onefuzz-file-format  = { path = "../onefuzz-file-format" }
```

Comment:
Instead of taking on this OneFuzz-specific crate as a dependency, can we instead:

1. Factor the `cobertura` module out of `onefuzz-file-format`, into its own crate
2. Make both `onefuzz-file-format` and `coverage` depend on the new `cobertura` crate

It's my fault for not doing it that way in the first place, since Cobertura is _not_ a OneFuzz-specific file format.